src/main/java/com/faforever/api/data/domain/BanType.java

```diff
@@ -0,0 +1,5 @@
+package com.faforever.api.data.domain;
+
+public enum BanType {
```

Comment:
Now I see why you named the other class `BanDurationType`. Maybe `BanContext` would be more precise here?